environment-model/environment-model_shacl.ttl

```diff
@@ -1,136 +1,159 @@
-            sh:message "Validation of creation_version failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
-            sh:name "creation_version"@en ;
-            sh:path environment-model:environment-model_project_creation_version ],
-        [ skos:example "high" ;
-            sh:datatype xsd:string ;
-            sh:description "category of the level of detail (High - highest level of detail with additional object enrichment, Med - directly from data sources, with environment, Low - topological representation)."@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of detail_level failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
-            sh:name "detail_level"@en ;
-            sh:path environment-model:environment-model_quality_detail_level ],
-        [ skos:example "12000" ;
-            sh:datatype xsd:int ;
-            sh:description "Total number of all geoemtries, instances are considered only once"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of geometry_count failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
-            sh:name "geometry_count"@en ;
-            sh:path environment-model:environment-model_quantity_geometry_count ],
-        [ skos:example "512m" ;
-            sh:datatype xsd:float ;
-            sh:description "tile size in meter"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of tile_size failed!"@en ;
-            sh:minCount "0"^^xsd:unsignedInt ;
-            sh:name "tile_size"@en ;
-            sh:path environment-model:environment-model_quantity_tile_size ],
-        [ skos:example "DataSmith" ;
+environment-model:environmentModelShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node general:GeneralShape ;
+            sh:order 1 ;
+            sh:path environment-model:general ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environment-model:QuantityShape ;
+            sh:order 2 ;
+            sh:path environment-model:quantity ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environment-model:QualityShape ;
+            sh:order 3 ;
+            sh:path environment-model:quality ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environment-model:ProjectShape ;
+            sh:order 4 ;
+            sh:path environment-model:project ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node environment-model:FormatShape ;
+            sh:order 5 ;
+            sh:path environment-model:format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 0 ;            
+            sh:node environment-model:GeoreferenceShape ;
+            sh:order 6 ;
+            sh:path georeference:georeference ] ;
+    sh:targetClass environment-model:environmentModel .   
```

Comment:
change this target class after ontology change

Reply:
fixed